lib/Jinja2Filters/strftime.py

```diff
@@ -0,0 +1,49 @@
+from isodatetime.parsers import (TimePointParser)  # Put in path by cylc.
```

Comment:
Please add standard header.

Should not need to have brackets around `(TimePointParser)`?